spot-client/src/spot-tv/app-state/actions.js

```diff
@@ -537,9 +536,7 @@ export function updateSpotTVSource() {
 export function redirectToMeeting(meetingNameOrUrl, { invites, meetingDisplayName, screenshare, startWithVideoMuted }) {
     return (dispatch, getState) => {
         const state = getState();
-        const domainWhitelist = getMeetingDomainsWhitelist(state);
```

Comment:
I'm wondering if appending 'zoom.us' at the `getMeetingDomainsWhitelist` level, would make things simpler?

Reply:
One concern I had was always creating a new array, which could trigger a re-render if it's being passed in directly. I also worried about mutating the array and if that's unexpected. So I though this reselect approach would be nice. But I could munge the config while we're setting the default config. what do you think of that? So instead of at the `getMeetingDomainsWhitelist` level, do it during init. This will work easily as long as we dont support dynamic config values.

Reply:
Yes